rules/windows/process_creation/win_hack_koadic.yml

```diff
@@ -21,8 +21,8 @@ logsource:
     product: windows
 detection:
     selection1:
-        CommandLine:
-            - '*cmd.exe* /q /c chcp *'
+        CommandLine|contains:
+            - 'cmd.exe* /q /c chcp '
```

Comment:
better to detect cmd.exe as a ProcessName (:

Reply:
Applied